src/handlers/dhcp/DhcpStreamHandler.h

```diff
@@ -118,18 +118,24 @@ class DhcpMetricsBucket final : public visor::AbstractMetricsBucket
 
 class DhcpMetricsManager final : public visor::AbstractMetricsManager<DhcpMetricsBucket>
 {
-    TransactionManager<uint32_t, DhcpTransaction, std::hash<uint32_t>> _request_ack_manager;
+    typedef TransactionManager<uint32_t, DhcpTransaction, std::hash<uint32_t>> DhcpTransactionManager;
+    std::unique_ptr<DhcpTransactionManager> _request_ack_manager;
 
 public:
     DhcpMetricsManager(const Configurable *window_config)
         : visor::AbstractMetricsManager<DhcpMetricsBucket>(window_config)
     {
+        if (window_config->config_exists("xact_ttl_secs")) {
+            _request_ack_manager = std::make_unique<DhcpTransactionManager>(static_cast<uint32_t>(window_config->config_get<uint64_t>("xact_ttl_secs")));
```

Comment:
what happens if it's a string, is that possible?

Reply:
This would except the as invalid value on policy creation... but it is not a good practice to except on constructor... 

What do you think of creating a do that validation on StreamHandler (ex. DhcpStreamHandler) Start? but it would generate a dependency of StreamHandler (ex. DhcpStreamHandler) creating the transaction manager on MetricsManager class (ex. DhcpMetricsManager)